lib/FSi/DoctrineExtensions/Metadata/ClassMetadata.php

```diff
@@ -1,12 +1,14 @@
 <?php
 
 /**
- * (c) Fabryka Stron Internetowych sp. z o.o <info@fsi.pl>
+ * (c) FSi sp. z o.o. <info@fsi.pl>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
 
+declare(strict_types=1);
+
 namespace FSi\DoctrineExtensions\Metadata;
 
 class ClassMetadata extends AbstractClassMetadata
```

Comment:
i suspect that this class isn't used anywhere. this class was useful when metadata was a separate component and this class could be used as some generic metadata storage. i think we can remove it now as concrete metadata classes do not use it.

Reply:
Next PR.